diffrax/_adjoint.py

```diff
@@ -325,18 +325,19 @@ def loop(
 
 
 class DirectAdjoint(AbstractAdjoint):
-    """A variant of [`diffrax.RecursiveCheckpointAdjoint`][]. The differences are that
-    `DirectAdjoint`:
-
-    - Is less time+memory efficient at reverse-mode autodifferentiation (specifically,
-      these will increase every time `max_steps` increases passes a power of 16);
-    - Cannot be reverse-mode autodifferentated if `max_steps is None`;
-    - Supports forward-mode autodifferentiation.
-
-    So unless you need forward-mode autodifferentiation then
-    [`diffrax.RecursiveCheckpointAdjoint`][] should be preferred.
-    If you need only forward-mode autodifferentiation, then [`diffrax.ForwardMode`][] is
-    more efficient.
+    """A variant of [`diffrax.RecursiveCheckpointAdjoint`][] that is also able to
+    support forward-mode autodifferentiation, whilst being less computationally
+    efficient. (Under-the-hood it is using several nested layers of `jax.lax.scan`s and
+    `jax.checkpoint`s, so that the cost of the solve increases with `max_steps`, even
+    if you don't need that many steps to perform the solve in practice.)
+
+    !!! warning
+
+        This method is not recommended! In practice you should almost always use either
+        [`diffrax.RecursiveCheckpointAdjoint`][] or [`diffrax.ForwardMode`][], depending
+        on whehter you need reverse or forward mode autodifferentiation. As this method
```

Comment:
"whehter" -> "whether" if we're being picky :)

Reply:
We absolutely are being picky :) Ty fixed!